hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkDataSet.java

```diff
@@ -829,13 +830,14 @@ public static DataSet toSpliceLocatedRow(JavaRDD<Row> rdd, OperationContext cont
                     context.getOperation().getExecRowDefinition().schema());
 
             String[] partitionByCols = new String[partitionBy.length];
+            StructField[] fields = context.getOperation().getExecRowDefinition().schema().fields();
```

Comment:
GetExecRowDefinition() is called on lines 830 and 833, so there is some unnecessary overhead.  Could we save the schema the first time and re-use it at line 833?